.circleci/config.yml

```diff
@@ -8,6 +8,7 @@ stripe_sdk_version: &stripe_sdk_version 13.2.0
 store_fabric_sdk_version: &store_fabric_sdk_version echo "$FABRIC_SDK_VERSION" > fabric_version.txt
 store_opentok_version: &store_opentok_version echo "$OPENTOK_VERSION" > opentok_version.txt
 store_stripe_version: &store_stripe_version echo "$STRIPE_SDK_VERSION" > stripe_version.txt
+preload_simulator: &preload_simulator xcrun instruments -w "iPhone 8 (12.1) [" || true
```

Comment:
Added this as a variable so that when we need to increment the simulator version we only need to change it in one place.

Reply:
Can't we use existing `&iphone_name` and add new constant `&iOS_version`?

Reply:
I don't think yaml works like that unfortch 🤔 https://stackoverflow.com/a/5954748/2271759